packages/chakra-ui/src/components/ui/checkbox.tsx

```diff
@@ -0,0 +1,19 @@
+import { Checkbox as ChakraCheckbox } from '@chakra-ui/react';
+import * as React from 'react';
```

Comment:
Two things. First, put the react imports up top. Secondly, don't use the `import * as React from 'react';` pattern. Instead import only that which you need. Like:
`import { InputHTMLAttributes, ReactElement, Ref, forwardRef } from 'react'`;